assertj-core/pom.xml

```diff
@@ -19,14 +19,11 @@
     <argLine>
       -Dfile.encoding=${project.build.sourceEncoding}
       -Dnet.bytebuddy.experimental=true
-      --add-opens=java.base/java.lang=ALL-UNNAMED
-      --add-opens=java.base/java.math=ALL-UNNAMED
-      --add-opens=java.base/java.util=ALL-UNNAMED
-      --add-opens=java.base/sun.nio.fs=ALL-UNNAMED
+      @${project.basedir}/argFile
```

Comment:
Kudos to @lobaorn and @sormuras – this seems to play nicely with both Maven and the IDE when on Windows 🙂

I hope you don't mind being mentioned as co-authors when I finally merge the changes 😉